deploy/app/legacyclaims.tf

```diff
@@ -0,0 +1,82 @@
+variable "legacy_claims_table_name" {
+  description = "The name of the DynamoDB table used by the legacy content-claims service"
+  type = string
+  default = "prod-content-claims-claims-v1"
+}
+
+variable "legacy_claims_bucket_name" {
+  description = "The name of the S3 bucket used by the legacy content-claims service"
+  type = string
+  default = "prod-content-claims-bucket-claimsv1bucketefd46802-1mqz6d8o7xw8"
+}
+
+variable "legacy_block_index_table_name" {
+  description = "The name of the legacy block index DynamoDB table"
+  type = string
+  default = "ep-v1-blocks-cars-position"
+}
+
+data "aws_s3_bucket" "legacy_claims_bucket" {
+  bucket = var.legacy_claims_bucket_name
+}
+
+data "aws_dynamodb_table" "legacy_claims_table" {
+  name = var.legacy_claims_table_name
+}
+
+data "aws_dynamodb_table" "legacy_block_index_table" {
+  name = "${terraform.workspace == "prod" ? "prod" : "staging"}-${var.legacy_block_index_table_name}"
+}
+
+data "aws_iam_policy_document" "lambda_legacy_dynamodb_get_document" {
+  statement {
+    actions = [
+      "dynamodb:GetItem",
```

Comment:
```suggestion
      "dynamodb:Query",
```

Reply:
I don't believe either of these tables are accessed using `GetItem`. You might want to rename `_get` in these definitions to `_query`?